@navikt/core/react/src/copybutton/CopyButton.tsx

```diff
@@ -119,6 +125,25 @@ export const CopyButton = forwardRef<HTMLButtonElement, CopyButtonProps>(
       }, activeDuration);
     };
 
+    const CopyIcon = () => {
+      return active ? (
+        <span className="navds-copybutton__icon">
+          {activeIcon ?? (
+            <CheckmarkIcon
+              aria-hidden={!!text}
+              title={text ? undefined : activeTitle}
+            />
+          )}
+        </span>
+      ) : (
+        <span className="navds-copybutton__icon">
+          {icon ?? (
+            <FilesIcon aria-hidden={!!text} title={text ? undefined : title} />
+          )}
+        </span>
+      );
+    };
```

Comment:
Ingen big deal, men `<span>` er repetert, og det er muligens litt dårlig praksis å definere komponenter inni komponenter. Forslag:

```suggestion
    const CopyIcon = (
      <span className="navds-copybutton__icon">
        {active
          ? activeIcon ?? (
              <CheckmarkIcon
                aria-hidden={!!text}
                title={text ? undefined : activeTitle}
              />
            )
          : icon ?? (
              <FilesIcon
                aria-hidden={!!text}
                title={text ? undefined : title}
              />
            )}
      </span>
    );
```